src/command_modules/azure-cli-backup/azure/cli/command_modules/backup/_help.py

```diff
@@ -115,16 +135,28 @@
 helps['backup policy list-associated-items'] = """
 type: command
 short-summary: List all items protected by a backup policy.
+examples:
+  - name: List all items protected by a backup policy
+    text: az backup policy list-associated-items --name MyBackupPolicy --resource-group MyResourceGroup --vault-name MyVault
+    crafted: true
 """
 
 helps['backup policy set'] = """
 type: command
 short-summary: Update the properties of the backup policy.
+examples:
+  - name: Update the properties of the backup policy. (autogenerated)
+    text: az backup policy set --policy {policy} --resource-group MyResourceGroup --vault-name MyVault
+    crafted: true
 """
 
 helps['backup policy show'] = """
 type: command
 short-summary: Show details of a particular policy.
+examples:
+  - name: Show details of a particular policy
```

Comment:
seeing this is already in, would we want to update crafted examples to have their names suffixed with " (autogenerated) ". 

cc: @glGarg @tjprescott

Reply:
Sorry about this. I will submit a separate PR to fix this in the next sprint (I've submitted too many PR's for this sprint already). Thanks!

Reply:
No worries, was just leaving a comment for the future!